src/client/themes/dark/index.ts

```diff
@@ -107,8 +107,14 @@ const darkTheme: DefaultTheme = {
       primary: '#00A3FF',
       onPrimary: '#FFFFFF',
       loading: '#FFA800',
-      error: '#FF005E',
-      warning: '#FFA800',
+      error: {
```

Comment:
we have this too, as global vars in default file. Don't know if it should change between themes or not, I will leave it at your criteria
![Uploading image.png…]()

Reply:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/78362532/164567629-4bccf9e3-ea52-4435-b29f-26f3841702bb.png">

Reply:
But those are for alerts, this is the modal, should we use the same for both?

Reply:
Idk, depending if the colors are ok, in reality that colors are just global variables, not only for alerts itself. It depends if the error colors should change betweeen themes I suppose

Reply:
For ease sake, keep it the same. After release (because I already freaking love this update to the errors) we can update UI for cleaner/easier reading. Function first IMO.